spi/common/web-spi/src/main/java/org/eclipse/edc/web/spi/configuration/WebServiceSettings.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (c) 2020-2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.web.spi.configuration;
+
+import org.eclipse.edc.web.spi.WebServer;
+
+public interface WebServiceSettings {
```

Comment:
Why introducing an interface and implementation? This could be a data class bringing information about a web service, there's no logic here, poliphormism is not really needed.